website/webpack.plugin.js

```diff
@@ -5,6 +5,9 @@ module.exports = function () {
     name: "docusaurus-webpack-plugin",
     configureWebpack() {
       return {
+        // There seems to be a rspack bug.
+        cache: false,
```

Comment:
Has this issue been reported to the docusaurus or rspace repo?

Reply:
Not yet, the error message is pretty irrelevant.
I can't reproduce it now, so let's leave it as is.